applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c

```diff
@@ -41,6 +41,8 @@ static struct bt_le_ext_adv *adv;
 static bool initialized;
 static bool delete_broadcast_src;
 
+uint8_t channel_mask[CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT];
```

Comment:
Please change to a descriptive name or add a comment

Reply:
Use audio_mapping_mask